data/strings/types_strings.txt

```diff
@@ -13296,7 +13136,7 @@
     zh-Hant = 樹林
 
   [type.landuse.forest.mixed]
-    en = Mixed Forest
+    en = Forest
```

Comment:
Not an improvement.   Just "Forest" is fine for unspecified, or if evergreen and coniferous also show as just "forest". But if this distinction is ever made, keep mixed.

Reply:
i still think removing mixed would be good. without the context that it's referring to the leaf cycle, it makes no sense. By contrast, explicit terms like evergreen are immediately obvious

Reply:
It's really a question of how much to project down to simpler logical tags.   But I see it as inconsistent to have deciduous/evergreen in text if tagged, and not show mixed.   But "Mixed deciduous/evergreen forest" is probably too lengthy.  Part of the issue is that these are somewhat technical terms and people who are into this sort of thing will understand and others will not understand.  But that's generally true, and I generally prefer to be correct for those that understand and not worry about those that don't finding it a bit odd.   However I realize "nerds-first design' is not the standard UX approach!

Reply:
fair, maybe Mixed-Leaf Forest will do?

Reply:
That's great.   It will be understood by tree nerds and normies alike and is short.  To me it's all about multiple kinds of audiences understanding what was meant.